src/inverseRobot.js

```diff
@@ -23,8 +23,26 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  let newRobot = {};
+  const checkValues = [];
```

Comment:
you don't need this array, just check that `newRobot` has no such property before adding it there